src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,66 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int MAX_LENGTH = 10;
```

Comment:
```suggestion
    private static final int MAX_LENGTH = 10;
```
коли у нас змінна використовується тільки в класі її потрібно робити приватною